index.js

```diff
@@ -33,6 +33,18 @@ function normalize(type, value) {
   return result;
 }
 
+function conforms(type, value) {
+  if (typeof type === 'string') {
+    return typeof value === type;
+  }
+  if (typeof type === 'function') {
+    return Boolean(type(value));
+  }
+  return type.some(function(type) {
```

Comment:
`some` is part of ES 5.1, which Node 0.10 fully supports as far as I know. I'll check.